### PR TITLE
Separate modules, rename functions

### PR DESCRIPTION
**Pull request by vacantron**

### CODE REVIEW

File: src/components/ToDoListContainer/ItemsList/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect, useCallback } from 'react';
+// import PropTypes from 'prop-types';
+import ToDoListItem from '../ToDoListItem/index';
+
+const resolveIndex = (objArr, id) => {
+  for (let index = 0; index < objArr.length; index++) {
+    if (objArr[index].id === id) return index;
+  }
+};
```
Comment (JiaAnTW) [inline src/components/ToDoListContainer/ItemsList/index.js:9]: https://developer.mozilla.org/zh-TW/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex

File: src/components/ToDoListContainer/ItemsList/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect, useCallback } from 'react';
+// import PropTypes from 'prop-types';
+import ToDoListItem from '../ToDoListItem/index';
+
+const resolveIndex = (objArr, id) => {
+  for (let index = 0; index < objArr.length; index++) {
+    if (objArr[index].id === id) return index;
+  }
```
Comment (JiaAnTW) [inline src/components/ToDoListContainer/ItemsList/index.js:8]: https://developer.mozilla.org/zh-TW/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex

**Review by JiaAnTW — APPROVED** [review]

File: src/components/ToDoListContainer/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+function ToDoListContainer() {
+  const [dataReceived, setDataReceived] = useState();
+
+  const handleItemAdd = (newItem) => {
+    setDataReceived(newItem);
+  };
+
+  return (
+    <Container>
+      <ToDoListInputForm handleItemAdd={handleItemAdd} />
```
Comment (JiaAnTW) [inline src/components/ToDoListContainer/index.js:15]: 這裡應該可以直接綁setDataReceived?

File: src/components/ToDoListContainer/ItemsList/index.js (hunk 2 above)
Comment (vacantron) [reply src/components/ToDoListContainer/ItemsList/index.js:8]: got it

**Pull request merged**